pages/dkp/konvoy/2.2/storage/static-local-volume/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: layout.pug
+navigationTitle: Provision a static local volume
+title: Provision a static local volume
+menuWeight: 15
+excerpt: Learn how to provision a static local volume for a Konvoy cluster
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+The `localvolumeprovisioner` addon uses the [local volume static provisioner][localstorage] to manage persistent volumes for pre-allocated disks.
+It does this by watching the `/mnt/disks` folder on each host and creating persistent volumes in the `localvolumeprovisioner` storage class for each disk that it discovers in this folder.
+
+- Persistent volumes with a 'Filesystem' volume-mode are discovered if they are mounted under `/mnt/disks`.
+
+- Persistent volumes with a 'Block' volume-mode are discovered if a symbolic link to the block device is created in `/mnt/disks`.
+
+# Before you begin
+
+Before starting this tutorial, verify the following:
+
+- You have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You have a provisioned `konvoy` cluster that uses the `localvolumeprovisioner` addon, but have not added any other addons to the cluster yet.
+
+  For this tutorial, you **do not deploy** using all the default settings as described in the [Quick start][quickstart].
+
+  This distinction between provisioning and deployment is important because some addons depend on the storage class provided by the `localvolumeprovisioner` addon and can fail to start if it is not configured yet.
+
+## Provision the cluster and a volume
+
+1. Provision the Kubernetes cluster without deploying addons by running the following command:
+
+    ```bash
+    konvoy provision
```

Comment:
This isn't the correct command in 2.1/2.2, is it?

Reply:
@dkoshkin or @faiq do you know if these commands apply to 2.1/2.2?

Reply:
Not a command. I believe Dimitri gave a suggestion with some updates